relay/prover.go

```diff
@@ -379,6 +461,26 @@ func (pr *Prover) ProveHostConsensusState(ctx core.QueryContext, height exported
 	return pr.originProver.ProveHostConsensusState(ctx, height, consensusState)
 }
 
+// shouldUseSHFUGRPC determines whether to use SHFU gRPC server based on config and environment variable
+// Environment variable YRLY_LCP_SHFU_GRPC_ENABLE=yes enables gRPC, disabled by default
+func (pr *Prover) shouldUseSHFUGRPC() (bool, string) {
+	// First check if address is configured
+	if pr.config.ShfuGrpcAddress == "" {
+		// No address configured, cannot use gRPC
+		return false, ""
+	}
+
+	// Check if gRPC is enabled via environment variable
+	envEnable := os.Getenv("YRLY_LCP_SHFU_GRPC_ENABLE")
+	if b, err := strconv.ParseBool(envEnable); err != nil || !b {
+		// Environment variable is not true or is not set, gRPC disabled by default
+		return false, ""
+	}
```

Comment:
Is this environment variable only used for testing purpose?
If so, it should be default to use gRPC if `shfu_grpc_address` is not empty.
In other words, this variable should be changed to YRLY_LCP_SHFU_GRPC_**DISABLE**.